ziti/src/main/kotlin/org/openziti/net/dns/ZitiDNSManager.kt

```diff
@@ -48,7 +48,7 @@ internal object ZitiDNSManager : DNSResolver, CoroutineScope {
         val ip = when {
             IPAddress.isValidIPv4(hostname) -> Inet4Address.getByName(hostname)
             IPAddress.isValidIPv6(hostname) -> Inet6Address.getByName(hostname)
-            else -> host2Ip.getOrPut(hostname) { nextAddr(hostname) }
+            else -> host2Ip.getOrPut(hostname.lowercase()) { nextAddr(hostname) }
         }
         launch {
             dnsBroadCast.emit(DNSResolver.DNSEvent(hostname, ip, false))
```

Comment:
why not emit the 'normalized' hostname and thus the test below wouldn't need to have been fixed?

Reply:
@ekoby - what happens for unqualified domain names? should we append a period to these too?

Reply:
I left the emit alone to minimize changes.  I can update the patch to 'fix' the emit too if that would be helpful.